CONTRIBUTING.md

```diff
@@ -71,7 +71,7 @@ make test
 
 #### Integration tests
 
-The entire test suite can take >20 mins to run, so we recommend against running them all locally. These will run on GitHub actions.
+The entire test suite can take ~7 mins to run, so we recommend against running them all locally. These will run on GitHub actions.
```

Comment:
Do we still recommend against running the whole thing locally?  It takes 2-3 minutes on my machine.